ElfUtils/ElfFile.cpp

```diff
@@ -89,6 +90,16 @@ void ElfFileImpl<ElfT>::InitSections() {
       has_debug_info_section_ = true;
     }
 
+    if (name.str() == ".text") {
+      llvm::Expected<llvm::ArrayRef<uint8_t>> text_section_contents =
+          elf_file->getSectionContents(&section);
+      if (!text_section_contents) {
+        LOG("Unable to get text section content");
+      } else {
+        text_section_ = text_section_contents.get();
```

Comment:
it does not seem to be used atm?

Reply:
Sorry for the mess. It is removed again in the latest commit. The rebase before merge will remove all traces of this.